flink-job/src/main/java/com/ververica/field/dynamicrules/functions/RuleDeserializer.java

```diff
@@ -29,7 +29,7 @@
 @Slf4j
 public class RuleDeserializer extends RichFlatMapFunction<String, Rule> {
 
-  private RuleParser ruleParser;
+  private transient RuleParser ruleParser;
```

Comment:
Seems like IntelliJ is using some advanced AI ;)

Reply:
that AI in particular sat in front of the screen ;)